### ISSUE TITLE
Highlight area on the image the AI identified an element to be checked

### ISSUE THREAD

**Issue by SantosSi** [openfoodfacts/openfoodfacts-hungergames#418]
Epic: user UI?

### Description
As a: user of the hunger games
I want: to see the area on the image the AI found an element to be checked
So that: I can easily visually compare it with the AI proposition and thus speed up the checking process.

### Acceptance criteria
A (configurable) highlight on the image where the AI found the element to be checked. 

### What would a demo look like
See Acceptance criteria.

### Notes
* next iteration's opportunity: ... so that: I see that part of the image zoomed in

**Comment by alexfauquette** [comment]
I would like to see this feature too, but the predictions are mostly based on text, and for human it's easier to spot a logo than a small piece of text

@alexgarel This may be linked to your idea of understanding how decisions are made

**Comment by alexgarel** [comment]
Yes it could be good to have this to understand what happens in robotoff.

But in hunger game I'm not sure, because it may leads toward more error, as bias from robotoff may transfer to human (eg. you will see the same word that leads to category decision, missing maybe other indices that's it's a different one).

**Comment by alexfauquette** [comment]
Yes, it would be relevant only when logo detector is used for labels questions
